docs/AdvancedStackatraceFeatures.md

```diff
@@ -0,0 +1,21 @@
+# Advanced Stacktrace Features
+
+## Display JIT compilation task
```

Comment:
What about `vtable` feature?

We can copy all relevant information from the corresponding issues (including screenshots) here and then refer to this chapter instead of github issues.

Reply:
Addressed in [e6282c8](https://github.com/async-profiler/async-profiler/pull/1029/commits/e6282c83bc0aa6007df310fd5e638e9d4d5ed6f1)